tests/openedx_tagging/core/tagging/test_views.py

```diff
@@ -637,34 +697,56 @@ def test_retrieve_object_tags_unauthorized(self):
         assert response.status_code == status.HTTP_403_FORBIDDEN
 
     @ddt.data(
-        (None, "abc", status.HTTP_401_UNAUTHORIZED, None),
-        ("user", "abc", status.HTTP_200_OK, 20),
-        ("staff", "abc", status.HTTP_200_OK, 20),
+        (None, "abc", status.HTTP_401_UNAUTHORIZED),
+        ("user_1", "abc", status.HTTP_200_OK),
+        ("staff", "abc", status.HTTP_200_OK),
     )
     @ddt.unpack
     def test_retrieve_object_tags_taxonomy_queryparam(
-        self, user_attr, object_id, expected_status, expected_count
+        self, user_attr, object_id, expected_status,
     ):
         """
         Test retrieving object tags for specific taxonomies provided
         """
+        object_id = "html7"
```

Comment:
nit: don't need the `object_id` as a param to this test if we're just going to hardcode it here.